### PR TITLE
Completed Css two column layout

### PR DESCRIPTION
**Pull request by vkahlon**
Screen Shot 2021-12-03 at 9 50 07 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  width: 50%;
+}
+
+div.title {
+  font-size: x-large;
+  letter-spacing: 2px;
+  margin-top: 5vh;
+}
+
+div.backend {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:28]: What is the reasoning for the name `backend` for this class?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One note for you:

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (vkahlon) [reply css-two-column-layout/styles.css:28]: Not an ideal name! But this div class working behind the scenes and took care of the dirty work. Ill resubmit it with a different name.

Comment (vkahlon) [reply css-two-column-layout/styles.css]: Replace backend name with background

**Review by codingcodymiller — APPROVED** [review]
Great work on this!!

**Pull request merged**